app/services/normalise_charge.js

```diff
@@ -145,20 +146,21 @@ module.exports = (function () {
 
   const apiPayload = function (req, card) {
     const payload = {
-      'card_number': creditCard(req.body.cardNo),
-      'cvc': req.body.cvc,
-      'card_brand': card.brand,
-      'expiry_date': expiryDate(req.body.expiryMonth, req.body.expiryYear),
-      'cardholder_name': req.body.cardholderName,
-      'card_type': card.type,
-      'corporate_card': card.corporate,
-      'prepaid': card.prepaid,
-      'address': addressForApi(req.body),
-      'accept_header': req.header('accept'),
-      'user_agent_header': req.header('user-agent')
```

Comment:
it seems my editor removed all `'`.
Let me know if it's an issue and I should roll it back.

Reply:
I think this would be good to do ^ 

I approve of removing the `'` but it should be done as a separate PR so that it's clear what has actually changed for this work.

Reply:
Sure - will clean it up.

Reply:
Nope, it's not my editor, it's pre-commit hook (linter) that "fixes" the files. Should I fight it?

Reply:
⚔️ 👍